.github/workflows/ci-docker-all-in-one.yml

```diff
@@ -23,8 +23,10 @@ jobs:
         mode:
         - name: v1
           binary: all-in-one
+          skip_healthcheck: true
         - name: v2
           binary: jaeger
+          skip_healthcheck: false
```

Comment:
please don't add this to the matrix. I know it's convenient but GitHub has an annoying pattern of merging all matrix values into the official name of the CI check such that when it's changed the `required` checks on the PR are no longer satisfied, as you can see happening in this PS:
<img width="975" alt="image" src="https://github.com/user-attachments/assets/45fba564-c15c-408b-9045-617e7a58d7d8">

Instead, we should really only have v1/v2 in the matrix and drive the other fields in the `run:` code with an if statement (but don't remove `binary`, for the same reason as above).

Reply:
Noted, will make the change.
But, I am not able to understand why the healthcheckV2 condition is failing because locally the test is working fine and the health is also being reported at localhost:13133/status

Reply:
The test is building a Docker image and testing against it. Since you added a new port you need to expose that port in the Docker command, otherwise it's not accessible.

Reply:
Also, you need to make sure that the extension server is running on the right IP, not on `localhost:13133`, because localhost inside the container is not reachable from outside the container.

Reply:
OK, will look into that.